interface/weno/facilities.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+/*
+ *  @package OpenEMR
+ *  @link    http://www.open-emr.org
+ *  @author  Sherwin Gaddis <sherwingaddis@gmail.com>
+ *  @copyright Copyright (c) 2020 Sherwin Gaddis <sherwingaddis@gmail.com>
+ *  @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+require_once("../globals.php");
+
+use OpenEMR\Common\Csrf\CsrfUtils;
+use OpenEMR\Core\Header;
+use OpenEMR\Rx\Weno\FacilityProperties;
+
+$data = new FacilityProperties();
+
+if ($GLOBALS['weno_rx_enable']) {
+    $data->ifcolumexist();
+}
```

Comment:
database changes should be in database.sql and the sql upgrade script (we should not be adding things on the fly since then to hard to keep track of the database schema)